src/python/WMCore/MicroService/MSPileup/MSPileupData.py

```diff
@@ -113,37 +114,58 @@ def createPileup(self, pdict):
             return [err.error()]
         return []
 
-    def updatePileup(self, doc):
+    def updatePileup(self, doc, rseList=None, validate=False):
```

Comment:
I thought that default should always be `validate=True` and only upstream code will explicitly change this parameter.

Reply:
Just for convenience, given that with this default, there is only 1 place calling it that needs to be changed (MSPileup). While the inverted logic would require changes in a handful of calls from MSPileupTasks (and maybe other places). Other than that, I have no strong preference and I could revert the logic if you want. Just let me know.

Reply:
Regardless how much changes should be done, the default should be right here. I strongly suggest to make `validate=True` as default and make explicit calls when this should be skipped.